mesheryctl/internal/cli/root/config/config.go

```diff
@@ -53,15 +54,44 @@ func (mc *MesheryCtlConfig) CheckIfCurrentContextIsValid() (Context, error) {
 	if mc.CurrentContext == "" {
 		return Context{}, errors.New("current context not set")
 	}
+
 	if ctx, exists := mc.Contexts[mc.CurrentContext]; exists {
 		if ctx.Version == "" {
 			ctx.Version = "latest"
 		}
+
+		version, err := CheckIfCurrentVersionIsValid(ctx.Version)
+		if err != nil || version == "" {
+			return Context{}, errors.New("invalid version " + ctx.Version + " specified")
+		}
+		ctx.Version = version
+		fmt.Println("The version for this context is: ", ctx.Version)
+
 		return ctx, nil
 	}
+
 	return Context{}, errors.New("current context:" + mc.CurrentContext + " does not exist")
 }
 
+// CheckIfCurrentVersionIsValid checks if current version is valid, return context as a string
+func CheckIfCurrentVersionIsValid(version string) (string, error) {
```

Comment:
I think we don't need to return the version again, just an `error` would do
We can make this a member function of the `Context` struct